examples/webserver.rs

```diff
@@ -27,25 +27,24 @@ fn main() -> Result<()> {
     };
 
     let camera = Camera::new(
-        peripherals.pins.gpio32,
-        peripherals.pins.gpio0,
-        peripherals.pins.gpio5,
-        peripherals.pins.gpio18,
-        peripherals.pins.gpio19,
-        peripherals.pins.gpio21,
-        peripherals.pins.gpio36,
-        peripherals.pins.gpio39,
-        peripherals.pins.gpio34,
-        peripherals.pins.gpio35,
-        peripherals.pins.gpio25,
-        peripherals.pins.gpio23,
-        peripherals.pins.gpio22,
-        peripherals.pins.gpio26,
-        peripherals.pins.gpio27,
+        /* PWDN */ peripherals.pins.gpio33, // Adjust X as needed
```

Comment:
this willl break compatibility with the esp32-cam, and only for the webserver example

Reply:
Sorry, these changes weren't meant to be included, they are for the specific board I am using, wrover dev